### DISCUSSION TITLE
Picture in Picture video player

### DISCUSSION THREAD

**Discussion by Flankiste** [FreeTubeApp/FreeTube#5846] [General]
Hi, i'm a big fan of Opera's overlay video player interface because it's the only one alowing you to navigate on the timeline of the vid. I found it particulary usfull when using ytb as a ressource  when coding or doing reaseaches while taking notes.
As i know there is no chromium extension or anything to replicate this behavour on other browser. It would be a great feature on FreeTube I think.

![image_2024-10-09_212711612](https://github.com/user-attachments/assets/62ca9c3f-47b5-483b-b9d1-1f40ed1cd3aa)

**Comment by absidue** [comment]
FreeTube uses the same PiP window as you get in your browser, so we can't control what is shown in the PiP window, as mentioned in many other issues.

**Reply by Flankiste** [reply]
No it's clearly not the same. And yes opera is my default browser.

![image_2024-10-10_143831479](https://github.com/user-attachments/assets/2965c178-5a73-48dd-b207-1a676090eb09)

**Reply by absidue** [reply]
Opera doesn't use the default Chromium PiP window they have heavily modified the Chromium one, which is why theirs looks like that but other Chromium based apps, like FreeTube, use the normal Chromium one.
